README.md

```diff
@@ -1,9 +1,16 @@
 <!-- ix-docs-ignore -->
 # imgex
+=======
+
```

Comment:
This doesn't render quite correct. We should only use a single style of markdown header.
```suggestion
```